build_vm_compilers/CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 ## 1.0.9-dev
 
 - Update to build version `2.0.0`.
+- Migrate to null safety
```

Comment:
It can compile files with null safety, but the generated dills don't run with sound null safety (unless I'm missing a flag). I've added a small note to clarify this.

Reply:
Ah ya its the sound/unsound distinction that it doesn't actually support then. I expected initially to need to do some additional stuff around the package config but actually build_modules is likely doing this.